src/fairchem/core/datasets/oc22_lmdb_dataset.py

```diff
@@ -114,9 +116,6 @@ def __init__(self, config, transform=None) -> None:
         if self.train_on_oc20_total_energies:
             with open(config["oc20_ref"], "rb") as fp:
                 self.oc20_ref = pickle.load(fp)
-        if self.config.get("lin_ref", False):
-            coeff = np.load(self.config["lin_ref"], allow_pickle=True)["coeff"]
-            self.lin_ref = torch.nn.Parameter(torch.tensor(coeff), requires_grad=False)
         self.subsample = aii(self.config.get("subsample", False), bool)
```

Comment:
this functionality is technically now supported with ur base dataset right? if so we can remove.